cmd/cli.go

```diff
@@ -184,6 +188,11 @@ var runcRestoreCmd = &cobra.Command{
 var restoreCmd = &cobra.Command{
 	Use:   "restore",
 	Short: "Manually restore a process from a checkpoint located at input path",
```

Comment:
Change the short description in the parent commands to give a hint for the options? Something like "manually checkpoint [process, runc (container), containerd (container)]"

Reply:
Oh shoot good catches thanks